### ISSUE TITLE
Finished torrents in Transmission still not removed

### ISSUE THREAD

**Issue by jamaroney** [barbequesauce/Watcher3#209]
Since latest commit didn't work at all for me using Transmission (3.0), I then updated to fdf8787 commit, yet finished torrents in Transmission still aren't showing as finished and thus not removed:

INFO [2023-02-13 10:53:17,916] CPTaskScheduler._task.273: == Finished Task: Torrents Status Check ==
INFO [2023-02-13 10:53:17,911] core.scheduler.check_torrents.391: No, did not find the torrent as finished
INFO [2023-02-13 10:53:17,910] core.scheduler.check_torrents.386: Check if we know finished torrent c38a37af02b9bb32689521cb9253f5ff45ebeac6 and is postprocessed (b'You.People.2023.720p.NF.WEBRip.800MB.x264-GalaxyRG[TGx]')
INFO [2023-02-13 10:53:17,910] core.scheduler.check_torrents.384: Torrent status c38a37af02b9bb32689521cb9253f5ff45ebeac6 [finished] (b'You.People.2023.720p.NF.WEBRip.800MB.x264-GalaxyRG[TGx]'); removetorrents: True
INFO [2023-02-13 10:53:17,898] core.downloaders.Transmission.get_torrents_status.151: Get torrents from transmissionrpc
INFO [2023-02-13 10:53:17,897] CPTaskScheduler._task.248: == Executing Task: Torrents Status Check Per User Command ==
INFO [2023-02-13 10:53:17,897] core.ajax.manual_task_execute.1464: Manually executing task Torrents Status Check.`
